packages/liveblocks-react-lexical/src/liveblocks-plugin-provider.tsx

```diff
@@ -249,17 +246,19 @@ export const LiveblocksPlugin = ({
         }}
       />
 
-      <CollaborationPlugin
-        // Setting the key allows us to reset the internal Y.doc used by useYjsCollaboration
-        // without implementing `reload` event
-        key={room.id}
-        id={room.id}
-        providerFactory={providerFactory}
-        username={username}
-        cursorColor={cursorcolor}
-        cursorsContainerRef={containerRef}
-        shouldBootstrap={true}
-      />
+      {self && (
+        <CollaborationPlugin
```

Comment:
`CollaborationPlugin` is only rendered once `self` is available. This is to work around Lexical's `CollaborationPlugin` issues around inability to update `username` and `cursor` after being set once 😭